source/manual/get-started.html.md

```diff
@@ -105,6 +107,40 @@ Some processes are dependent on your local gov.uk setup being in `~/govuk`:
     mac$ cd ~/govuk
     mac$ git clone git@github.com:alphagov/govuk-puppet.git
 
+### 3.1 Ensure your Ruby setup works
+
+You'll want to install the [specific version of Ruby in govuk-puppet][https://github.com/alphagov/govuk-puppet/blob/master/.ruby-version#L1] (currently `rbenv install 1.9.3-p550`), as the default system Ruby is too locked-down.
+
+Now verify that rbenv is working as expected:
+
+    mac$ cd ~/govuk/govuk-puppet
+    mac$ rbenv versions
+
+This should output something like:
+
+```
+  system
+* 1.9.3-p550 (set by /Users/yourname/govuk/govuk-puppet/.ruby-version)
+```
+
+    mac$ which ruby
+
+This should output something like:
+
+```
+/Users/yourname/.rbenv/shims/ruby
+```
+
+If, instead, this outputs `/usr/bin/ruby`, then you'll need to update your `~/.bash_profile` to have `ruby` properly overridden in your `PATH`:
+
+```bash
+# force load rbenv to ensure correct version of ruby used
+export PATH="$PATH:~/.rbenv/shims"
+if which rbenv > /dev/null; then eval "$(rbenv init -)"; fi
+```
```

Comment:
These instructions _look_ like overkill given we installed with Homebrew, but I had to follow these steps myself to get the setup to work persistently!